SPECS/cyrus-sasl-bootstrap/cyrus-sasl-bootstrap.spec

```diff
@@ -25,7 +25,6 @@ Requires:       openssl
 Requires:       pam
 Requires:       systemd
 Requires:       libdb
-AutoProv:       no
```

Comment:
Please remove the other `AutoProv` in this spec as well. In general, you can pretty much revert all since the most recent commit from this file. I'd only keep the new release number.